test/spmd/checkpoint/test_fsdp_optim_state.py

```diff
@@ -14,13 +14,13 @@
 
 from spmd.testing.checkpoint_utils import with_temp_dir
 from spmd.testing.common_utils import (
-    TEST_GPU_NUM,
     DistTensorTestBase,
     with_comms,
 )
 from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
 from torch.testing._internal.common_utils import run_tests
 
+TEST_GPU_NUM = 4
```

Comment:
Looks like there is a decorator named "skip_unless_torch_gpu", maybe we can directly use that?